members.js

```diff
@@ -1,3 +1,48 @@
-const members = [];
+const members = [
+  {
+    firstName: "John",
+    lastName: "Doe",
+    jobTitle: "Software Engineer",
+    department: "Engineering",
+    manager: null,
+    location: "Glasgow",
+    firstAider: "No",
+    userSlack: "johndoe",
+    timeZone: "America/Los_Angeles",
+  },
+  {
+    firstName: "Jane",
+    lastName: "Doe",
+    jobTitle: "Software Engineer",
+    department: "Engineering",
+    manager: "John Doe",
+    location: "Leeds",
+    firstAider: "No",
+    userSlack: "janedoe",
+    timeZone: "America/Los_Angeles",
+  },
+  {
+    firstName: "Bob",
+    lastName: "Doe",
+    jobTitle: "Software Engineer",
+    department: "Engineering",
+    manager: "John Doe",
+    location: "Cape Town",
+    firstAider: "No",
+    userSlack: "bobdoe",
+    timeZone: "America/Los_Angeles",
+  },
+  {
+    firstName: "Sally",
+    lastName: "Doe",
+    jobTitle: "Software Engineer",
+    department: "Engineering",
+    manager: "John Doe",
+    location: "Rome",
+    firstAider: "No",
```

Comment:
Nice work, I used boolean for ``firstAider`` key but it's not a big deal at all

Reply:
I thought about that, but then I decided not to modify the given data as it says that we will be updating it later.  Thank you for taking the time to go through it. @SamanZahedi